### ISSUE TITLE
Add --disable-open-browser option on the airshipui binary command line

### ISSUE THREAD

**Issue by aschiefe** [airshipit/airshipui#14]
Octant allows for the starting of the binary without popping the browser:
```
$ octant --disable-open-browser
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "deployment/configuration", "module-name": "overview"}
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/containerEditor", "module-name": "overview"}
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/serviceEditor", "module-name": "overview"}
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/startPortForward", "module-name": "overview"}
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/stopPortForward", "module-name": "overview"}
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/commandExec", "module-name": "overview"}
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/deleteTerminal", "module-name": "overview"}
2020-01-29T21:14:16.172Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "octant/deleteObject", "module-name": "configuration"}
2020-01-29T21:14:16.198Z        INFO    plugin/manager.go:397   registered plugin "argo-ui"     {"plugin-name": "argoui", "cmd": "/root/.config/octant/plugins/argoui", "metadata": {"Name":"argo-ui","Description":"Argo UI version e93fe60","Capabilities":{"IsModule":true}}}
2020-01-29T21:14:16.198Z        INFO    plugin/manager.go:405   plugin supports navigation      {"plugin-name": "argoui"}
2020-01-29T21:14:16.198Z        INFO    dash/dash.go:391        Using embedded Octant frontend
2020-01-29T21:14:16.204Z        INFO    dash/dash.go:370        Dashboard is available at http://127.0.0.1:7777
```
Airship UI does not which results in an error showing when you have to tunnel the connection over ssh for the airshipui:
```
$ bin/airshipui
2020-01-29T21:15:44.246Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/deleteTerminal", "module-name": "overview"}
2020-01-29T21:15:44.246Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "deployment/configuration", "module-name": "overview"}
2020-01-29T21:15:44.246Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/containerEditor", "module-name": "overview"}
2020-01-29T21:15:44.246Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/serviceEditor", "module-name": "overview"}
2020-01-29T21:15:44.247Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/startPortForward", "module-name": "overview"}
2020-01-29T21:15:44.247Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/stopPortForward", "module-name": "overview"}
2020-01-29T21:15:44.247Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "overview/commandExec", "module-name": "overview"}
2020-01-29T21:15:44.247Z        INFO    module/manager.go:79    registering action      {"component": "module-manager", "actionPath": "octant/deleteObject", "module-name": "configuration"}
2020-01-29T21:15:44.275Z        INFO    plugin/manager.go:397   registered plugin "argo-ui"     {"plugin-name": "argoui", "cmd": "/root/.config/octant/plugins/argoui", "metadata": {"Name":"argo-ui","Description":"Argo UI version e93fe60","Capabilities":{"IsModule":true}}}
2020-01-29T21:15:44.275Z        INFO    plugin/manager.go:405   plugin supports navigation      {"plugin-name": "argoui"}
2020-01-29T21:15:44.276Z        INFO    dash/dash.go:391        Using embedded Octant frontend
2020-01-29T21:15:44.284Z        INFO    dash/dash.go:370        Dashboard is available at http://127.0.0.1:7777

2020-01-29T21:15:44.284Z        WARN    dash/dash.go:374        unable to open browser: exec: "xdg-open": executable file not found in $PATH
github.com/vmware-tanzu/octant/pkg/dash.(*dash).Run
        /go/src/github.com/vmware-tanzu/octant/pkg/dash/dash.go:374
github.com/vmware-tanzu/octant/pkg/dash.Run.func1
        /go/src/github.com/vmware-tanzu/octant/pkg/dash/dash.go:203
```

**Comment by aschiefe** [comment]
Octant does not have the command line options exposed so that we can reference them and their related accoutrement.  

https://github.com/vmware-tanzu/octant/issues/652

**Comment by dotNomad** [comment]
Closed by f884450fb0d4beaeaf678e5c1760c2c7d850f5fa
